TrustModel_Personas.md

```diff
@@ -0,0 +1,155 @@
+  [A Technical Analysis of Confidential Computing v1.1](https://confidentialcomputing.io/wp-content/uploads/sites/85/2021/03/CCC-Tech-Analysis-Confidential-Computing-V1.pdf)
+ defines Confidential Computing as the protection of data in use by performing computation in a 
+  hardware-based Trusted Execution Environment (TEE).
+
+In considering personas we recognise that a trust boundary exists between each persona and we
+explore how the least privilege principle (as described on Page 36 of
+  [Cloud Native Security Paper](https://github.com/cncf/tag-security/blob/main/security-whitepaper/CNCF_cloud-native-security-whitepaper-Nov2020.pdf)
+) should apply to any actions which cross these boundaries.
+
+ Confidential containers can provide enhancements to ensure that the expected code/containers 
+ are the only code that can operate over the data. However any vulnerabilities within this code 
+ are not mitigated by using confidential containers, the Cloud Native Security Whitepaper 
+ details Lifecycle aspects that relate to the security of the code being placed into containers 
+ such as Static/Dynamic Analysis, Security Tests, Code Review etc which must still be followed. 
+
+![Personas Model](./images/PersonaModel.svg)
+
+Any of these personas could attempt to perform malicious actions.
+
+### Infrastructure Operator
+This persona has privileges within the Cloud Infrastructure which includes the hardware and 
+firmware used to provide compute, network and storage to the Cloud Native solution
+They are responsible for availability of infrastructure used by the cloud native environment.
+- Have access to the physical hardware.
+- Have access to the processes involved in the deployment of compute/storage/memory used by any 
+  orchestration components and by the workload.
+- Have control over TEE hardware availability/type
+- Responsibility for applying firmware updates to infrastructure including the TEE Technology.
+
+Example : Cloud Service Provider (CSP) Infrastructure as a Service (IaaS) Site Reliability Engineer
+(SRE)
+
+### Orchestration Operator
+This persona has privileges within the Orchestration/Cluster.
+They are responsible for deploying a solution into a particular cloud native environment and 
+managing the orchestration environment.
+For managed cluster this would also include the administration of the cluster control plane.
+- control availability of service
+- control webhooks and deployment of workloads
+- control availability to workload of cluster resources (data/networking/storage) and cluster 
+  services (Logging/Monitoring/Load Balancing)
+- control the deployment of boot images initially required by the TEE during initialisation. 
+ These boot images once initialised will receive the workload.
+
+Example : A kubernetes administrator responsible for deploying pods to a cluster and 
+maintaining the cluster.
+
+### Workload Provider
+This persona develops and builds the code, containers and pods intended to provide the 
```

Comment:
I think this persona may be two separate personas.
There is one persona which creates and runs the solution. And I think a separate persona which creates one or more container images used in the solution. There is an important security relationship between them for our model. 
Specifically, the container image publisher signs and/or encrypts the image. 
The workload provider/owner/operator must trust the signature of the publisher and/or be authorized to decrypt the image.

Reply:
I agree that this Workload Provider is perhaps a summary persona, My intention is to submit a pull request once this one is merged which will look into the pipeline behind providing the workload, relating it the CNCF model and also starting to discuss where setting up attestation flows fits .
So I am keen not to break up the Workload Provider into more personas in this PR but I am happy to perhaps highlight fact that behind the workload provider sit multiple personas that fulfill the actions listed?
If that helps?

Reply:
I think recognizing at least one degree of separation is fundamental to our security model and should be defined up front. If we set aside the potentially complex set of actors in k8s we still have a message (a container image) which is signed and/or encrypted. And there are usually at least two parties, the signer and the verifier (and similarly an actor that encrypts and one that decrypts).

Because a workload may be a composition of multiple container images from multiple authors there is a further complication. I'm not concerned about defining that upfront as much as being clear about this distinction of who signs/encrypts an individual container and who verifies/decrypts an individual container.

Reply:
OK I see what you mean, 
I want to keep the personas for this level of documentation to be larger grained, but I agree that opening the door to the distinction between who signs/verifies and who encrypts makes a lot of sense. It also helps setup the more detailed conversations and exploration of threats to come.
So I have reworked things a little and introduced the container image provider persona to help explore this separation of signing/encrypting from verifying. Hopefully this captures the point you were making?